src/pages/settings/Wallet/PaymentMethodList.tsx

```diff
@@ -193,12 +180,18 @@ function PaymentMethodList({
     style = {},
     listItemStyle = {},
     shouldShowRightIcon = true,
+    invoiceTransferBankAccountID,
 }: PaymentMethodListProps) {
     const styles = useThemeStyles();
     const StyleUtils = useStyleUtils();
     const {translate} = useLocalize();
     const {isOffline} = useNetwork();
     const [isUserValidated] = useOnyx(ONYXKEYS.USER, {selector: (user) => !!user?.validated});
+    const [bankAccountList = {}] = useOnyx(ONYXKEYS.BANK_ACCOUNT_LIST);
+    const [cardList = {}] = useOnyx(ONYXKEYS.CARD_LIST);
```

Comment:
We actually don't need cards here, just bank accounts, so we can remove this

Reply:
The app uses this component for the wallet page as well. So I don't think we can safely remove `cardList`.

https://github.com/Expensify/App/blob/53a11d7291d9d35252c9534628dba459e6d0a8a3/src/pages/settings/Wallet/PaymentMethodList.tsx#L213-L216

Reply:
Ok, let's set it as empty then to make sure cards don't show up there and leave a comment like 

// We only allow bank accounts for invocing transfers so we're making the cardlist empty here.

Reply:
Or do you think we should handle it differently?

Reply:
Actually, we already handled this case. 

1. There is a prop `shouldShowAssignedCards` of `PaymentMethodList`. This is `false` by default.

https://github.com/Expensify/App/blob/53a11d7291d9d35252c9534628dba459e6d0a8a3/src/pages/settings/Wallet/PaymentMethodList.tsx#L95-L96

https://github.com/Expensify/App/blob/53a11d7291d9d35252c9534628dba459e6d0a8a3/src/pages/settings/Wallet/PaymentMethodList.tsx#L189

2. There is a condition for `filteredPaymentMethods` to skip the `cardList` by the prop.

https://github.com/Expensify/App/blob/53a11d7291d9d35252c9534628dba459e6d0a8a3/src/pages/settings/Wallet/PaymentMethodList.tsx#L213-L216

Is it clear now?

Reply:
Ah ok, got it, thanks